### PR TITLE
Reenabling Elemental spell for Hellfire

### PR DESCRIPTION
**Pull request by thebigMuh**

### CODE REVIEW

**Comment by NikoVP** [comment]
Great news!

**Comment by galaxyhaxz** [comment]
Hmm, Elemental should never have been disabled in the first place, unless this was an accident carried over from the spawn build which removed many spells.

**Comment by thebigMuh** [comment]
> Hmm, Elemental should never have been disabled in the first place, unless this was an accident carried over from the spawn build which removed many spells.

Yeah I talked to AJenbo about this, and he thinks it may have been a mistake introduced during a merge.

**Comment by AJenbo** [comment]
could you fix this upstream?

File: Source/spells.cpp [hunk 1]
```diff
@@ -405,15 +396,6 @@ int GetSpellStaffLevel(spell_id s)
 	if (!gbIsHellfire && s > SPL_LASTDIABLO)
 		return -1;
 
-	if (gbIsHellfire) {
-		switch (s) {
-		case SPL_ELEMENT:
```
Comment (AJenbo) [inline Source/spells.cpp:410]: please add it to the spawn exception

**Review by AJenbo — APPROVED** [review]

File: Source/spells.cpp (hunk 1 above)
Comment (thebigMuh) [reply Source/spells.cpp:410]: Did so. I tried finding a comprehensive list of changes between standard Diablo and the Spawn install, to check for other differences, but wasn't really able to find anything. Do you maybe have a link?

**Pull request merged**